packages/rum/src/domain/record/serialization/serializeNode.spec.ts

```diff
@@ -150,11 +134,7 @@ describe('serializeNodeWithId', () => {
     })
 
     it('serializes attributes', () => {
-      const element = document.createElement('div')
-      element.setAttribute('foo', 'bar')
-      element.setAttribute('data-foo', 'data-bar')
-      element.className = 'zog'
-      element.style.width = '10px'
+      const element = append('<div foo="bar" data-foo="data-bar" class="zog" style="width: 10px;"></div>')
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏this was ensuring that browsers return the changed attributes even if they are set with specific setters instead of `setAttribute`.

It could be nice to keep it this way:

```ts
const element = append('<div foo="bar" data-foo="data-bar"></div>')
element.className = 'zog'
element.style.width = '10px'
```